notebooker/web/routes/scheduling.py

```diff
@@ -52,6 +52,8 @@ def remove_schedule(job_id):
 
 
 def get_job_id(report_name: str, report_title: str) -> str:
+    if "PATH_TO_CATEGORY_DICT" in current_app.config and report_name in current_app.config["PATH_TO_CATEGORY_DICT"]:
```

Comment:
I don't think this way of retrieving the category is tenable - it assumes the webapp still has access to the template files (which is not guaranteed) and that get_directory_structure() has run (which is not guaranteed at this point as far as I know)